File: gateOne/dayThree/java/TaskOne.java
import java.util.Random;
import java.util.Scanner;
import java.util.ArrayList;
public class TaskOne{
	public static void main(String... args){
	Scanner scanner = new Scanner(System.in);

	ArrayList <Integer> correct = new ArrayList<>();
	ArrayList <Integer> incorrect = new ArrayList<>();
		int number = 0;
		for(int counter = 0; counter < 5; counter++){
		Random randomNum1 = new Random();
			int showMe = randomNum1. nextInt(10);
		Random randomNum2 = new Random();
			int newOne =  randomNum2.nextInt(10);

			System.out.print(showMe+ "x"+newOne +" = ");
				int num = scanner.nextInt();
				if (num == showMe * newOne){
					correct.add(num);
				}
				else{
					incorrect.add(num);
				}
								
			System.out.print(showMe+ "+" +newOne+" = ");
				int sum = scanner.nextInt();
				if (sum == showMe + newOne){
					correct.add(sum);
				}
				else{
					incorrect.add(sum);
				}

		}
		
		System.out.println("The total correct score is : "+correct.size());
		System.out.println("The total incorrect score is :"+ incorrect.size());

	}
}

File: gateOne/dayThree/phyton/task_one.py
import random
correct =[]
incorrect =[]
for counter in range(5):
	num = random. randint(1, 10)
	nums = random.randint(1, 10)
	
	first_calc = int(input(f"{num} x {nums} = "))
	if (first_calc == num * nums):
		correct.append(first_calc)
	else:
		incorrect.append(first_calc)
	second_calc = int(input(f"{num} + {nums} = "))
	if(second_calc == num + nums):
		correct.append(second_calc)
	else:
		incorrect.append(second_calc)
print(f"The total correct scores is {len(correct)}")
print(f"The total incorrect scores is {len(incorrect)}")
	
	
